plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -115,6 +115,11 @@ public class DefaultJdbcMetadata
 
     private final SyntheticColumnHandleBuilder syntheticColumnBuilder;
 
+    public DefaultJdbcMetadata(JdbcClient jdbcClient, boolean precalculateStatisticsForPushdown, Set<JdbcQueryEventListener> jdbcQueryEventListeners)
```

Comment:
I'm thinking whether we should fight for consistency here in terms of the next constructor below having line breaks on the params and this one not.